src/solver/optimisation/adequacy_patch.cpp

```diff
@@ -27,49 +27,107 @@
 
 #include "../simulation/simulation.h"
 #include "adequacy_patch.h"
+#include <map>
 
-using namespace Antares;
-using namespace Antares::Data;
-using namespace Antares::Data::AdequacyPatch;
+namespace Antares
+{
+namespace Data
+{
+namespace AdequacyPatch
+{
```

Comment:
I do not think this is what his comment wants. For me, he wants to remove "using namespace Antares::Data::AdequacyPatch", then write "AdequacyPatch::adqmPhysicalAreaInsideAdqPatch" instead of "adqmPhysicalAreaInsideAdqPatch".
Again, I do not think what he wants is more clear than the previous version.

Reply:
Reading the comments and not really sure any more. 
It's a small change so we will resolve it tomorrow meeting.

Reply:
I pushed a version only using the Antares::Data namespace, but I agree it looks more messy now.